modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/css/recycle_bin/RecycleBin.scss

```diff
@@ -0,0 +1,30 @@
+.recycle-bin-section {
+	.recycle-bin-title {
```

Comment:
I think most of this styles can be replaces with some utility classes. 
 
Check https://clayui.com/docs/components/text for font-sizes, weights and colors (`text-3`, `font-wight-semi-bold `or `text-secondary` are some examples)

Reply:
You can use the class `.text-dark` where needed and remove this scss file.